docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+<!--[metadata]>
++++
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_content_trust"
+weight=-1
++++
+<![end-metadata]-->
+
+# Content trust in Docker
+
+When transferring data among networked systems, *trust* is a central concern.
```

Comment:
Maybe elaborate a bit more?

When transferring data among networked systems, *trust* is a central concern. In particular, when communicating over an untrusted medium such as the internet, it is critical to ensure the integrity and origin of the all the data a system operates on.

When using Docker you push and pull images (data) to a registry. Content trust gives you the ability to both verify the integrity and the publisher of all the data received from a registry over any channel. This feature is currently only available for users of Docker's public Hub, but it will be also made available for the Docker Trusted Registry or for private registries in the near future.